app/src/main/java/com/rootstrap/android/network/managers/ManagerModule.kt

```diff
@@ -7,11 +7,11 @@ import com.rootstrap.android.network.managers.user.UserManagerImpl
 import dagger.Binds
 import dagger.Module
 import dagger.hilt.InstallIn
-import dagger.hilt.android.components.ApplicationComponent
+import dagger.hilt.components.SingletonComponent
 import javax.inject.Singleton
 
 @Module
-@InstallIn(ApplicationComponent::class)
+@InstallIn(SingletonComponent::class)
```

Comment:
could you please explain this change?

Reply:
ApplicationComponent was renamed to SingletonComponent to allow the usage of Hilt in non-Android modules. You can check this release notes for further information : https://github.com/google/dagger/releases/tag/dagger-2.28.2